### PR TITLE
Renames non-cybernetic implants to bio-chips

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Renames non-cybernetic implants to bio-chips
Renames non-cybernetic implant implanters to bio-chip implanters

## Why It's Good For The Game
The amount of times you have to specify what sort of implant you are talking about is crazy, this changes that with a non-conflicting item name change.
It makes 100% more sense to have these two groups under different names.
Please for the love of god do not make me continue having to specify cybernetic vs non-cybernetic when discussing implants.

## Testing
It's all string changes heaven help me if i mised one

## Changelog
:cl:
tweak: non-cybernetic implants are now called bio-chips
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
contra might be renaming cybernetic implants instead, i dont know.

**Comment by Contrabang** [comment]
microchips are what are in your food, not what macrobombs are made of

**Comment by Carthusia** [comment]
I like the idea of a rename to make the separation a bit more clear, but im not afan of Microchip.

Calling something like a nutrient implant a microchip just feels wrong, since its not a small implant, a microchip is something like a mindslave implant, so tiny that not passive and active scans can bring it up.

I feel a more techy name would be more fitting, like say shadowruns various augmentation types fit into these category's:

Cyberware
Bioware
Nanoware
Geneware
Chemware

**Comment by TORE01** [comment]
How about nanomachines?

**Comment by Carthusia** [comment]
It still has the issue of a name that implies its way way way smaller than the item actually is, Micro is like a computer chip, nano is almost cellular in size.

I think ive also mis-interpreted what your aiming for here in terms of name changes, i thought you we're changing the names of other implant types, like surgical implants to make it clearer where they go.

For something like a mindsheild implant, mindslave implant and chemical implants im not sure what the issue is?

Could you give me an example of the kind of situation where this is confusing people?

**Comment by procdrone** [comment]
i proposed this before, maybe you can find some informative propositions https://www.paradisestation.org/forum/topic/21709-rename-implants-to-something-else/

**Comment by S34NW** [comment]
changed to bio-chip

**Comment by Miraviel** [comment]
I am personally against this idea, there are a lot of "bio-chips" now that make no sense to be a chips. Microbombs, macrobombs, dusters, adrenal implants, EMP implants, these all need more than a chip that messes with your body/brain. While "implant" can be confusing for the RND implants, "bio-chip" would be just as confusing how a tiny processor in your brain blows up entire cargo upon death. Currently, they would be "chips" because you inject them with an injector. It would clear up the RND vs non-RND confusion and would create new ones.

I suggest taking the longer route here and make a list of every possible implants and properly categorize them into a) bio-chip b) implant, where bio-chips are things affecting your brain or muscles (mindslave, anti-drop, CNS rebooter, etc.) while implants have reagents in them or do something sci-fi like dusting (emp, adrenal, nutriment, bomb, duster).

**Comment by procdrone** [comment]
> I am personally against this idea, there are a lot of "bio-chips" now that make no sense to be a chips. Microbombs, macrobombs, dusters, adrenal implants

A very simple explanation. these devices aren't actually implanted into the body, but are rather bluespace chips that call in these devices upon activation and take immediate effect (like bombs)

Just like uplink is a rather small device and can use telecrystals to teleport in rather huge equipment, chips could be beacons for this stuff.

besides, I do feel like other "implants" make much more sense to be called chips. like mindshield, or death implant.

The truth is, we can explain this any way anyone wants, the salient point is, to differentiate skin implants from organ implants. not everything makes perfect sense, and doesn't need to. (like clown having chance to close a singularity down), etc.

**Comment by Sirryan2002** [comment]
if you guys are looking for a "lore" explanation for how implants can physically exist within a person as a biochip, you can always add that info to the implant pad.

**Comment by S34NW** [comment]
image
☠️

File: code/game/objects/items/weapons/implants/implantpad.dm [hunk 1]
```diff
@@ -58,19 +58,19 @@
...
-/obj/item/implantpad/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = TRUE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
-	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
-	if(!ui)
-		ui = new(user, src, ui_key, "ImplantPad", name, 410, 400, master_ui, state)
-		ui.open()
+/obj/item/implantpad/attackby(obj/item/implantcase/C as obj, mob/user as mob, params)
+	if(istype(C, /obj/item/implantcase))
+		addcase(user, C)
+	else
+		return ..()
```
Comment (hal9000PR) [inline code/game/objects/items/weapons/implants/implantpad.dm:73]: accidental revert?

File: code/game/machinery/cloning.dm [hunk 2]
```diff
@@ -626,7 +626,7 @@ GLOBAL_LIST_INIT(cloner_biomass_items, list(\
 	Using the H-87 is almost as simple as brain surgery! Simply insert the target humanoid into the scanning chamber and select the scan option to create a new profile!<br>
 	<b>That's all there is to it!</b><br>
 	<i>Notice, cloning system cannot scan inorganic life or small primates.  Scan may fail if subject has suffered extreme brain damage.</i><br>
-	<p>Clone profiles may be viewed through the profiles menu. Scanning implants a complementary HEALTH MONITOR IMPLANT into the subject, which may be viewed from each profile.
+	<p>Clone profiles may be viewed through the profiles menu. Scanning implants a complementary HEALTH MONITOR BIOCHIP into the subject, which may be viewed from each profile.
```
Comment (farie82) [inline code/game/machinery/cloning.dm:629]:
```suggestion
	<p>Clone profiles may be viewed through the profiles menu. Scanning implants a complementary HEALTH MONITOR BIO-CHIP into the subject, which may be viewed from each profile.
```
For consistency with the rest

File: code/game/objects/items/weapons/implants/implantpad.dm (hunk 1 above)
Comment (S34NW) [reply code/game/objects/items/weapons/implants/implantpad.dm]: Should be okay now @hal9000PR

**Review by hal9000PR — APPROVED** [review]
chippy time

**Pull request merged**